src/main/java/org/broadinstitute/dsm/model/at/DefaultValues.java

```diff
@@ -37,6 +37,7 @@
             if (participantDataList == null) {
                 Map<String, Object> esParticipantData = entry.getValue();
                 Map<String, Object> profile = (Map<String, Object>) esParticipantData.get(ElasticSearchUtil.PROFILE);
+//                TODO only for `registration_type` = `self` && `dependent`
```

Comment:
@gmakharat can you add that check in for me.
DSM should only create the genomicId if pt has in ddp_participant_data at field_type_id: `AT_GROUP_MISCELLANEOUS`
`REGISTRATION_TYPE` = `Self` OR `Dependent`